src/mixins/editorMixin.js

```diff
@@ -86,10 +86,34 @@ export default {
 				onOutlineToggle: (visible) => {
 					this.toggleOutlineFromEditor(visible)
 				},
+				onLoaded: () => {
+					console.log('onLoaded')
+				},
 			})
+
+			if (document.location.hash) {
+				window.setTimeout(() => {
+					const element = document.querySelector(document.location.hash)
+					const topPos = element.offsetTop
+
+					getScrollParent(element)?.scrollTo({
```

Comment:
Any reason why you don't just do `element?.scrollIntoView({ behavior: 'smooth' })` directly? I don't understand why you would want to get the parent node if `scrollHeight` is larger than `clientHeight` :thinking:

Reply:
I'm going to double check it one more time. But as far I can remember - there was an issue with sticky header overlapping

Reply:
Just checked again

`element?.scrollIntoView({ behavior: 'smooth' })`:
![image](https://github.com/nextcloud/collectives/assets/191082/682a6bcc-6b93-4b46-8b19-ed48b5b9d649)

Seems like we need to play with `scroll-margin-top` or something like that

Reply:
Soo, I've investigated it little bit.

We need add few lines to [NcAppComponent](https://github.com/nextcloud-libraries/nextcloud-vue/blob/master/src/components/NcAppContent/NcAppContent.vue#L432)
```css
  overflow-y: auto;
  scroll-padding-top: 60px;
  scroll-behavior: smooth;
```
This needed to fix scrolling when you clicking on anchors.

But it not helps to `scrollIntoView` :disappointed: . It scrolls more than needed, take a look on video

https://github.com/nextcloud/collectives/assets/191082/1f2ad586-2ad7-4b0e-9abf-f0537bdbf05f